modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2669,6 +2669,86 @@ TEST_P(Test_ONNX_layers, YOLOv7)
     normAssertDetections(refClassIds, refScores, refBoxes, keep_classIds, keep_confidences, keep_boxes);
 }
 
+
+TEST_P(Test_ONNX_layers, YOLOv5n)
+{
+    std::string weightPath = _tf("models/yolov5n.onnx", false);
```

Comment:
```suggestion
    std::string weightPath = findDataFile("dnn/yolov5n.onnx", false);
```

Reply:
There is a quite a number of test with `_tf` as reader. Which one to use in general ?

Reply:
They are locally defined methods to keep paths shorter. However this model is located in a different folder:
https://github.com/opencv/opencv/blob/7783c229771259e16e70c047af074c29c66eb324/modules/dnn/test/test_onnx_importer.cpp#L15C1-L20C1